### ISSUE TITLE
Configuring step on readme leads you in error

### ISSUE THREAD

**Issue by Zikoel** [cheat/cheat#597]
**A description of the problem**
If you follow the readme section `Configuring` you can go in a not well configured configuration. If you execute `mkdir -p ~/.config/cheat && cheat --init > ~/.config/cheat/conf.yml` you have a valid configuration but `cheat` doesn't create the `personal` or `community` folder but that folder provided from the default configuration are needed, so the app crash on the next run!

If you create the folders manully the app doesn't crash but the community repo is not cloned so you start without data.

If you don't execute the suggested command instead `cheat` guide you on the configuration steps, the config file is correctly created, the necessary folders are correctly created and the community repo il correctly pulled.

In conclusion not follow the `Configuring` step is better for a new user.

**cheat version info**
`4.1.1`

**Comment by chrisallenlane** [comment]
Hi, @Zikoel 

Thanks for bringing this to my attention.

I updated the `README` to be clearer, and removed the information regarding `cheat --init` entirely, given that users should no longer need it. I've also added instructions on how to install `cheat` using `go get`. Here's the diff:

https://github.com/chrisallenlane/cheat/commit/0718b606e1ebf1c32c2a203b71b3d17353200891

Please let me know if you have any further feedback.

Thanks!
